### DISCUSSION TITLE
How to add "mailto:" when binding and email address to :href

### DISCUSSION THREAD

**Discussion by bodigital** [alpinejs/alpine#2155] [1. Help] [answered]
Hi, I'm looping through a collection of email addresses, and would like the template to be a mailto-link. Do I have to create a separate function for this or is there an easy way around that I miss?

```
<div x-data="{emails: [{'type': 'personal', 'email': 'test@test.com'}, {'type': 'work', 'email': 'another@test.com'}]}">
  <template x-for="mail in emails">
    <a x-text="mail.email" :href="mail.email"
       class="inline-flex mb-2 items-center justify-center flex-shrink-0 w-full px-4 py-2 text-sm font-medium text-white bg-indigo-600 border border-transparent rounded-md shadow-sm hover:bg-indigo-700 sm:flex-1"></a>
  </template>
</div>
```

**Comment by danddanddand** [comment, marked as answer]
You just have to build the string in js. So for example changing your :href expression to "'mailto:' + mail.email" or making helper function that returns template literal.

**Reply by bodigital** [reply]
Thank you! I was unaware that I could simply manipulate the string the way you described in this scenario.
